sql/table.go

```diff
@@ -164,6 +164,8 @@ func encodeTableKey(b []byte, v parser.Datum) ([]byte, error) {
 		return encoding.EncodeNumericFloat(b, float64(t)), nil
 	case parser.DString:
 		return encoding.EncodeBytes(b, []byte(t)), nil
+	case parser.DNull:
+		return b, nil
```

Comment:
I wish we could put in something reasonable here. Do you think we should add in a unique encoding for NULL values to differentiate between secondary keys like ('a', NULL) and (NULL, 'a')